stubs/Pillow/PIL/ImageFont.pyi

```diff
@@ -1,16 +1,16 @@
-from _typeshed import SupportsRead
+from _typeshed import StrOrBytesPath, SupportsRead
 from typing import Any, Protocol
 from typing_extensions import Literal
 
 LAYOUT_BASIC: Literal[0]
 LAYOUT_RAQM: Literal[1]
 
 class _Font(Protocol):
-    def getmask(self, text: str, mode: str = ..., direction=..., features=...): ...
+    def getmask(self, text: str | bytes, mode: str = ..., direction=..., features=...): ...
```

Comment:
This declaration confused me; the code itself doesn't have these named parameters (but instead are just `*args` and `**kwargs`).